backend/prisma/migrations/20250624181240_remove_email_user/migration.sql

```diff
@@ -0,0 +1,15 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `email` on the `User` table. All the data in the column will be lost.
+  - A unique constraint covering the columns `[username]` on the table `User` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- DropIndex
+DROP INDEX "User_email_key";
```

Comment:
I'm not too familiar with these migration sql files but I remember you having them in earlier PRs. Are these re-usable if you need to recreate your database? If so then maybe to avoid database issues you should update the original sql file to have the right indexes and columns etc.